src/test/ui/nll/issue-47388.stderr

```diff
@@ -1,10 +1,12 @@
 error[E0594]: cannot assign to data in a `&` reference
   --> $DIR/issue-47388.rs:18:5
    |
-LL |     let fancy_ref = &(&mut fancy);
-   |                     ------------- help: consider changing this to be a mutable reference: `&mut`
 LL |     fancy_ref.num = 6; //~ ERROR E0594
    |     ^^^^^^^^^^^^^^^^^ `fancy_ref` is a `&` reference, so the data it refers to cannot be written
+help: consider changing this to be a mutable reference: `&mut (&mut fancy)`
+   |
+LL |     let fancy_ref = ;
```

Comment:
You can see what I'm referring to in the previous comments here: `let fancy_ref = ;` should be 

```
help: consider changing this to be a mutable reference
   |
LL |     let fancy_ref = &mut (&mut fancy);
```
`span_suggestion` deals with this correctly automatically.